lib/viewcomponents/port.jsx

```diff
@@ -103,8 +106,8 @@ var Port = React.createClass({
 
     this.setState({
       draggingPosition: {
-        x: lastPos.x + event.movementX,
-        y: lastPos.y + event.movementY
+        x: lastPos.x + event.movementX / this.props.parentScale,
```

Comment:
This ought to be done in the pseudo drag utility, so that `event.movementX` and `event.movementY` come pre-scaled. My original intent with that drag handler was to abstract the scale away.